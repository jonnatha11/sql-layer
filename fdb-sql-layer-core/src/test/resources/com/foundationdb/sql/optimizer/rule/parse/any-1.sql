SELECT order_date FROM orders
 WHERE 100 < ANY (SELECT price * quan FROM items 
                   WHERE items.oid = orders.oid)
