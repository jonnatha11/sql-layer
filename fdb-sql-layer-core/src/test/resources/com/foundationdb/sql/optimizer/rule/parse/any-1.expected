SelectQuery@50502819
  ResultSet@201d592a[order_date]
    Project@66c8e4de[orders.order_date]
      Select@6401d98a[ANY(Subquery@154e45b3)]
        TableSource@178aab40(orders)

Subquery@154e45b3
  Project@1fbbc779[100 < times(items.price,items.quan)]
    Select@10f102d3[items.oid == orders.oid]
      TableSource@3ef07355(items)
